microsoft/testsuites/network/networksettings.py

```diff
@@ -174,6 +174,17 @@ def validate_ringbuffer_settings_change(self, node: Node) -> None:
         priority=1,
     )
     def validate_device_channels_change(self, node: Node, log: Logger) -> None:
+        kernel_ver = node.tools[Uname].get_linux_information().kernel_version
+        if (
+            isinstance(node.os, Ubuntu)
+            and node.os.information.release <= "16.04"
+            and kernel_ver <= "4.13.15"
```

Comment:
this is string comparison not version. if you check "4.2" <= "4.13.15", it return False